packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -446,14 +446,20 @@ export class AppStudioPluginImpl {
       (isLocalDebug ? "local" : ctx.envInfo.envName) +
       `.json`;
     if (!(await fs.pathExists(manifestFileName))) {
-      return err(
-        AppStudioResultFactory.UserError(
-          AppStudioError.FileNotFoundError.name,
-          AppStudioError.FileNotFoundError.message(manifestFileName) +
-            " Run 'Provision in the cloud' first. Click Get Help to learn more about why you need to provision.",
-          HelpLinks.WhyNeedProvision
-        )
-      );
+      const isProvisionSucceeded = !!(ctx.envInfo.state
```

Comment:
Check whether it's already provisioned. If the project has provisioned, generate the package first.

The "teamsAppUpdatedAt" field is in state file, so git clone will not discard the information. It works well to check the timestamp of dev portal and local when updating manifest file.

Reply:
Looks good~